sdk-actors/src/test/java/io/dapr/actors/runtime/DaprInMemoryStateProvider.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.actors.runtime;
+
+import io.dapr.actors.ActorId;
+import io.dapr.client.DaprClient;
+import reactor.core.publisher.Mono;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class DaprInMemoryStateProvider extends DaprStateAsyncProvider {
+
+  private static final Map<String, byte[]> stateStore = new HashMap<>();
+
+  private final ActorStateSerializer serializer;
+
+  DaprInMemoryStateProvider(ActorStateSerializer serializer) {
+    super(null, serializer);
+    this.serializer = serializer;

```

Comment:
Is this defined 2x?  It is in the parent as well.

Reply:
Yes. I did not want to change visibility of that property just for the tests. Sometimes I change but this time is avoidable.

Reply:
It's confusing on first read and might be better if it did not have the same name as the hidden parent member.

Reply:
Passing null to super now.